_data/backup.yml

```diff
@@ -21,7 +21,8 @@ websites:
     img: backblaze.png
     tfa: Yes
     sms: Yes
-    doc: https://www.backblaze.com/blog/two-factor-verification-for-backblaze/
+    software: Yes
```

Comment:
sorry, forgot to copy and paste this link in the description

https://www.backblaze.com/blog/two-factor-verification-via-totp/